simple/msg_epoll.c

```diff
@@ -342,6 +348,9 @@ int main(int argc, char **argv)
 
 	fi_shutdown(ep, 0);
 	ft_free_res();
+#else
+	ret = -FI_ENOSYS;
+#endif
```

Comment:
Just #ifdef the entire file, with a new main that returns ENOSYS if epoll is not set.  ifdefs in the middle of C code is to be avoided as much as possible, especially within functions.